app/tests/components_tests/test_amazon_sagemaker_training_backend.py

```diff
@@ -196,6 +197,7 @@ def test_execute(settings):
                     "PYTHONUNBUFFERED": "1",
                     "no_proxy": "amazonaws.com",
                     "GRAND_CHALLENGE_COMPONENT_WRITABLE_DIRECTORIES": "/opt/ml/output/data:/opt/ml/model:/opt/ml/checkpoints:/tmp",
+                    "GRAND_CHALLENGE_COMPONENT_POST_CLEAN_DIRECTORIES": "/opt/ml/output/data:/opt/ml/model",
```

Comment:
Note to self - before deploying we will need to upgrade all of the active images to use the latest version of SageMaker Shim.